backend/app/services/jobs.py

```
@@ -0,0 +1,194 @@
+    """
+    try:
+        acquired = await cache_manager.setnx(
+            REPO_UPDATE_LOCK_KEY,
+            True,
+            expire=REPO_UPDATE_LOCK_KEY_INTERVAL
+        )
+        if acquired:
+            logger.info(f"[job] Successfully acquired distributed lock: {REPO_UPDATE_LOCK_KEY}")
+        else:
+            logger.info(f"[job] Failed to acquire distributed lock, lock is held by another instance: {REPO_UPDATE_LOCK_KEY}")
+        return acquired
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)